libs/layer-composer/src/generators/heatmap/heatmap-animated.ts

```diff
@@ -108,6 +113,39 @@ const serializeBaseSourceParams = (params: TileAggregationSourceParams) => {
   return serialized
 }
 
+const getFinalurl = (
+  config: GlobalHeatmapAnimatedGeneratorConfig,
+  params: TileAggregationSourceParamsSerialized
+) => {
+  const { datasets, filters, 'vessel-groups': vesselGroups, ...rest } = params
+  const finalUrlParams = {
+    ...rest,
+    format: 'intArray',
+    'temporal-aggregation': params.singleFrame === 'true',
```

Comment:
I think this is always false, params.singleFrame === 'true' only in non animated heatmap